src/viam/services/vision.py

```diff
@@ -1,10 +1,11 @@
 import json
 from dataclasses import dataclass
 from enum import Enum
+from io import BytesIO
 from typing import Any, List, Mapping, Optional, Sequence, Union
 
 from grpclib.client import Channel
-from PIL.Image import Image
+from viam.media.viam_rgba_plugin import Image
```

Comment:
This is just an alias for `from PIL import Image`. Not sure why you needed to import this (and then change all the `Image` definitions to `Image.Image`...

Reply:
@gvaradarajan says we need it to support the viam raw format (otherwise it isn't registered?)

Reply:
Yuuup I see